ramls/bound-with-part.raml

```diff
@@ -0,0 +1,48 @@
+#%RAML 1.0
+title: Bound-With API
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://github.com/org/folio/mod-inventory-storage
+
+documentation:
+  - title: Bound-With API
+    content: This documents the API calls that can be made to query and manage parts of bound-withs
+
+types:
+  boundWithPart: !include boundwithpart.json
+  boundWithParts: !include boundwithparts.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  pageable: !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+  language: !include raml-util/traits/language.raml
+  validate: !include raml-util/traits/validation.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/bound-with-parts:
+  type:
+    collection:
+      exampleCollection: !include examples/boundwithparts.json
+      exampleItem: !include examples/boundwithpart.json
```

Comment:
Personally I find it easier to read multi-word file names when there are hyphens between the words

Reply:
Yeah, I tried to follow what looked like the de-facto naming convention but I don't disagree with the preference.

Reply:
> Yeah, I tried to follow what looked like the de-facto naming convention but I don't disagree with the preference.

It's challenging because unless someone changes all of the examples at once (which folks don't have time to do), we only compound the (less readable IMO) convention

Reply:
Right, agreed, I changed it to hyphens.

Reply:
Thank you